### PR TITLE
Opensearch manifest 1.1.0

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
Opensearch manifest 1.1.0
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/1.1.0/manifest.linux.x64.1.1.0.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+manifest:
+  schema-version: 1.0
+build:
+  version: 1.1.0
+  platform: linux
+  architecture: x64
+products:
+#  opensearch:
```
Comment (dblock) [inline manifests/1.1.0/manifest.linux.x64.1.1.0.yml:9]: You should delete this and not comment out. We have git for history.

Comment (peterzhuamazon) [reply manifests/1.1.0/manifest.linux.x64.1.1.0.yml]: Removed.

File: manifests/1.1.0/manifest.linux.arm64.1.1.0.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+manifest:
+  schema-version: 1.0
+build:
+  version: 1.1.0
+  platform: linux
+  architecture: arm64
+products:
+#  opensearch:
```
Comment (abhinavGupta16) [inline manifests/1.1.0/manifest.linux.arm64.1.1.0.yml]: Can we remove it altogether?

Comment (peterzhuamazon) [reply manifests/1.1.0/manifest.linux.arm64.1.1.0.yml]: Removed

**Review by abhinavGupta16 — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]
nit: The manifest name should have been manifest.linux.arm64-1.1.0.yml
Too many dots is weird

**Pull request merged**